profileFunctions.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+/***********************************************
+ * Functions to pull content from Db to CMS page
+ ***********************************************/
+
+/**
+ * Gets profile text from Db
+ *
+ * @param $text string Gets profile text from Db
+ *
+ * @return array
+ */
+function getProfileText($db) {
+    $textQuery = $db->prepare("SELECT `text` FROM `profile`;");
+    $textQuery->execute();
+    $textArray = $textQuery->fetch();
+    return $textArray;
```

Comment:
no need to assign `$textQuery->fetch();` to a variable. Just `return $textQuery->fetch();`

Reply:
Good point. I'll look at it in later code refactoring.